examples/index/index.ts

```diff
@@ -1,15 +1,16 @@
 import basicEditor from '../../packages/bundles/basicEditor';
 import { jabberwocky } from '../utils/jabberwocky';
+import { MarkdownRenderer } from '../../packages/plugin-markdownrenderer/src/MarkdownRenderer';
 import { Youtube } from '../../packages/plugin-youtube/src/Youtube';
 import './index.css';
 
 jabberwocky.init(basicEditor.editable);
 
 basicEditor.loadConfig({
     debug: true,
-    // renderTo: 'markdown',
+    renderTo: 'markdown',
```

Comment:
@dmo-odoo
I'm facing a conundrum.

1. nodes are not aware of renderers
2. renderers are not aware of nodes
3. nodes render themselves
4. renderers handle generic steps of rendering (node creation, mapping, attributes)

-> Where do we create links?

Eg:
A markdown renderer must render Formats in a special way, therefore it must be
aware and able to extend anything that handles formats. But we could have
different plugins handling formats.

------
Also, do they all render ultimately to html to display in the DOM? Albeit
differently?

Eg:
VDocument
```html
<paragraph>
    <Char.b:a/>
</paragraph>
<paragraph>
    <Char:b/>
</paragraph>

```
HTMLRenderer
```html
<p><b>a</b></p>
<p>b</p>
```
MarkdownRenderer
```html
<p><b>*a*</b></p>
<p>b</p>
```
or
```html
<div>*a*<br>b</div>
```
...or 
```javascript
"*a*\nb"
```
?